tests/e2e/playwright.config.ts

```diff
@@ -13,8 +13,7 @@ configDotenv({
 });
 
 const vsix: string = globSync(path.resolve(__dirname, "..", "..", "out", "*.vsix")).at(0) as string;
-const isInsiders = process.env.TERM_PROGRAM_VERSION?.endsWith("insider");
-const vscodeVersion = process.env.VSCODE_VERSION || (isInsiders ? "insiders" : "stable");
```

Comment:
Unlike the mocha tests, we shouldn't be trying to match the to-be-tested VS Code version with the currently-running VS Code version.